CPC-CHARTER.md

```diff
@@ -237,27 +237,32 @@ CPC Chairperson or Director may serve.
 ## Section 8. Board Representation
 
 There are 3 Board seats earmarked for representation from the
-OpenJS Foundation projects.  The first of these board seats is filled by
-the CPC Director as outlined in Section 7 on [Elections][] and is
-always in place.  
+OpenJS Foundation projects.
 
-The second board seat requires 2 Platinum board members to activate
-and will be filled based on processes defined
+### CPC Director
+
+The first of these board seats is filled by the CPC Director as outlined in Section 7 on [Elections][] and is always in place.
+
+**CPC Director eligibility:** The CPC Director must be a current CPC Member
+
+**CPC Director voting:** CPC Members will vote on candidates
+
+### Community Director
+
+The second board seat will be filled based on processes defined
 by the CPC and approved by the Voting CPC members and then the Board. The
 CPC members will review and either re-confirm or update the
 processes once per year.
 
-For the first year after the formation of the OpenJS Foundation, the
-processes for filling this seat will be as follows:
+**Community Director eligibility:** The Community Director must be an [Active OpenJS Collaborator][] as defined in the CPC's Governance document.
 
-* As a constellation member of the OpenJS Foundation, the Node.js project
-  will elect one Board representative from either the Technical Steering Committee (TSC) or
-  Community Committee (CommComm). The representative will be selected according to a process
-  defined by the TSC and CommComm.
+**Community Director voting:** CPC Members will vote on candidates
 
 As described in the [Elections][] section a multiple candidate method
 will be be used for the voting for the first and second board members.
 
+### At-Large Director
+
 The third board seat is earmarked to represent the Individual membership
 program. 3 Platinum board members as well as 2000 members registered in the
 Individual membership program are required for it to activate.
```

Comment:
I would like to keep this name change separate as it does not relate to this PR.

Reply:
I guess it depends then on which PR lands first?

Reply:
That makes sense. It can be very confusing either way. I'm putting a pull request in today to move this to Stage 1 and complete the rename. Hopefully that can move this through to remove confusion.